src/card/models/printed_card.rb

```diff
@@ -2,9 +2,11 @@ module Card
   module Models
     class PrintedCard
       attr_reader :image_path, :spec
+      attr_accessor :image_url
 
-      def initialize(image_path, spec)
+      def initialize(image_path, spec, image_url=nil)
         @image_path = image_path
+        @image_url = image_url
```

Comment:
This class is overloaded- I'm going to revisit it in a future PR. I think that given that it's past tense ("Printed"), we should only have the image url attached.